HTML:CSS learning project/style.css

```diff
@@ -1,3 +1,31 @@
-h1 {
-  color: red;
+header{
```

Comment:
Selecting an element by a global tag in CSS is NOT a good practice. Try to implement classes for elements that need styles. For example, the header could have class 'main-header' and so on.

Your task here: refactor both files so there won't be any global selectors in the CSS file but the styles would remain the same.